geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/iterators/ArrowFileIteratorTest.scala

```diff
@@ -1,10 +1,10 @@
-/*******************************************************************************
+/***********************************************************************
 * Copyright (c) 2013-2017 Commonwealth Computer Research, Inc.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Apache License, Version 2.0
 * which accompanies this distribution and is available at
 * http://www.opensource.org/licenses/apache2.0.php.
-******************************************************************************/
+*************************************************************************/
```

Comment:
Ah, might run mvn license:check locally.  

Actually, bonus points if you can add that plugin to the verify lifecycle.